package/MDAnalysis/core/universe.py

```diff
@@ -1147,7 +1182,7 @@ def add_angles(self, values, types=None, guessed=False):
         .. versionadded:: 1.0.0
         """
         self._add_topology_objects('angles', values, types=types,
-                                 guessed=guessed)
+                                   guessed=guessed)
```

Comment:
```suggestion
                                 guessed=guessed)
```

Revert unrelated change.